prisma/migrations/20210404163725_added_star_lesson_relation/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AddForeignKey
+ALTER TABLE "stars" ADD FOREIGN KEY ("lessonId") REFERENCES "lessons"("id") ON DELETE SET NULL ON UPDATE CASCADE;
```

Comment:
Make the column not null, since every star needs a lesson. With that, deletion should also cascade instead of setting it to null.

Also give some stars on your dev db and then try applying the migration to see if doesn't cause any errors.

```suggestion
-- AddForeignKey
ALTER TABLE "stars" ADD FOREIGN KEY ("lessonId") REFERENCES "lessons"("id") ON DELETE CASCADE ON UPDATE CASCADE;

-- Make the column not nullable
ALTER TABLE "stars" ALTER COLUMN "lessonId" SET NOT NULL;
```